src/web/server/document.tsx

```diff
@@ -20,15 +22,37 @@ interface RenderToStringResult {
 export const document = ({ data }: Props) => {
     const { CAPI, NAV, linkedData } = data;
     const title = `${CAPI.headline} | ${CAPI.sectionLabel} | The Guardian`;
+    const pathToDist = process.env.NODE_ENV === 'production' ? '.' : 'dist';
+    // This is the stats file generated by webpack loadable plugin
+    const statsFile = path.resolve(`${pathToDist}/loadable-stats.json`);
```

Comment:
Might be clearer to just use an if statement with some repetition here rather than the ternary?